crates/roc_host/src/lib.rs

```diff
@@ -16,12 +16,6 @@ use std::time::{Duration, SystemTime, UNIX_EPOCH};
 use std::{env, io};
 use tokio::runtime::Runtime;
 
-/// Implementation of the host.
-/// The host contains code that calls the Roc main function and provides the
-/// Roc app with functions to allocate memory and execute effects such as
-/// writing to stdio or making HTTP requests.
```

Comment:
This comment was for the entire file, not just for `mod heap`. Perhaps rustfmt removed the lines in between the comments and the code...

Reply:
add in a983b2e0976521f8d43053b2ffc35a8fe69fab45, also checked agains the `cargo doc --open`

<img width="1034" alt="Screenshot 2024-09-29 at 09 35 41" src="https://github.com/user-attachments/assets/02defb5c-9823-45d1-b142-a17d1fd2a198">